### ISSUE TITLE
Add: Backup .jar for coding lib

### ISSUE THREAD

**Issue by DarkGuardsman** [BuiltBrokenModding/VoltzEngine#52]
Inject a .jar inside of VE to be used as a backup to be unpacked if coding lib can not be downloaded. This way VE can be installed offline and if the maven can not be accessed.

**Comment by DarkShadow44** [comment]
As of today, CodingLib can't be downloaded anymore, since "api.dmodoomsirius.me" is down.

**Comment by DarkGuardsman** [comment]
Yep, it has been down for a while now... this has been low priority as it was only used in the 1.7.10 versions of our mods. Which we stopped supporting a few years ago. Along with the entire Voltz Engine project which I killed off a when we moved to 1.12.

That said we have a backup file in our support channel https://discord.gg/MTcKRXv36G

**Comment by DarkGuardsman** [comment]
Since this is a common issue people ask I made a release on coding-lib github https://github.com/DarkGuardsman/CodingLib/releases/tag/0.2.8

**Comment by DarkShadow44** [comment]
Thanks you! It's still useful, because yours is literally the only functioning version of (calclavia) MFFS still around. It's a shame it died.
